micromamba/tests/test_create.py

```diff
@@ -281,8 +281,6 @@ def test_target_prefix(
     else:
         root_prefix = Path(os.environ["MAMBA_ROOT_PREFIX"])
 
-    # TODO: Remove this call to `os.makedirs` once
-    # https://github.com/mamba-org/mamba/issues/3790 is fixed
     if root_prefix_env_exists:
         os.makedirs(Path(os.environ["MAMBA_ROOT_PREFIX"]) / "envs", exist_ok=True)
 
```

Comment:
I guess you meant to do this?
```suggestion
```

Can you check that `root_prefix/envs` exists on the filesystem at the end of the test if `root_prefix_env_exists`?